File: code/stable/projectile/src/java/Projectile/OutputFormat.java
package Projectile;

import java.util.Arrays;
import java.util.BitSet;
import java.util.Scanner;
import java.io.PrintWriter;
import java.io.FileWriter;
import java.io.File;
import java.util.ArrayList;

public class OutputFormat {
    
    public static void write_output(String s, double d_offset) throws Exception {
        PrintWriter outputfile;
        outputfile = new PrintWriter(new FileWriter(new File("output.txt"), false));
        outputfile.print("s = ");
        outputfile.println(s);
        outputfile.print("d_offset = ");
        outputfile.println(d_offset);
        outputfile.close();
    }
}

File: code/stable/sglpend/src/cpp/OutputFormat.cpp
#include "OutputFormat.hpp"

#include <fstream>
#include <iostream>
#include <string>

using std::ofstream;
using std::string;

void write_output(double θ_p) {
    ofstream outputfile;
    outputfile.open("output.txt", std::fstream::out);
    outputfile << "θ_p = ";
    outputfile << θ_p << std::endl;
    outputfile.close();
}
